src/main/frontend/db/rules.cljc

```diff
@@ -136,6 +136,12 @@
                [(str ?val) ?str-val]
                [(contains? ?v ?str-val)]))]
 
+   :page-refs
+   '[(page-refs ?b ?page-name)
```

Comment:
Since `?page-name` can have multiple elements, worth pluralizing this to `?page-names`

Reply:
Yes, much better +1